src/components/ShortTextInput.jsx

```diff
@@ -0,0 +1,92 @@
+import React, {useState} from 'react'
+import { Link } from 'gatsby'
+import styled from "styled-components"
+
+const StyledShortTextInput = styled.section`
+    width: 37.5rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    padding: 1em 1em 2em 1em;
+
+  div {
+    width: 23rem;
+    /* height: 15rem; */
+    border: 1px solid var(--dark-grey);
+    border-radius: 2.2px;
+    padding: .2rem 1rem;
+    /* margin: 1rem 1rem 2rem 1rem; */
+  }
+  div:active {
+    border: 2px solid var(--button-outline-blue);
+  }
+
+  div:focus-within {
+    border: 2px solid var(--button-outline-blue);
+  }
+
+  label {
+    font-weight: 300;
+    font-size: .8em;
+    color: var(--dark-grey);
+    display: block;
+    margin-bottom: .1rem;
+  }
+  
+  input {
+    margin-left: .1rem;
```

Comment:
Looks like this is offsetting the text a bit. We can probably remove this line.

Reply:
the input area is offset to the right in the figma. let me know if i shouldn't be following that too strictly.